pkg/aws/core_test.go

```diff
@@ -289,6 +317,96 @@ var _ = Describe("MachineServer", func() {
 		)
 	})
 
+	Describe("#GetPlacementObject", func() {
```

Comment:
Can we also add one positive test case in this?

Reply:
done, thanks